ansible_collections/arista/avd/molecule/evpn_underlay_ebgp_overlay_ebgp/documentation/devices/DC1-BL1A.md

```diff
@@ -353,6 +354,13 @@ interface Ethernet4
    no switchport
    ip address 172.31.255.87/31
 !
+interface Ethernet7
+   description test
+   mtu 9000
+   no switchport
+   vrf Tenant_A_WAN_Zone
+   ip address 10.10.10.10/24
+!
```

Comment:
Missing no shutdown in generated config. I think you need to add `enabled: true`